docs/tutorials/fundamentals/getting_started.md

```diff
@@ -47,14 +49,19 @@ If the image is `RGB` or `RGBA` use the `-r` or `--rgb` flag.
 
 ![image: napari viewer displaying an image layer](../assets/tutorials/launch_cli_image.png)
 
-Launching napari directly from the command line is the simplest and fastest way to open the viewer,
+Launching `napari` directly from the command line is the simplest and fastest way to open the viewer,
 but it doesn't allow you to preprocess your images before opening them.
 It is also currently not possible to save images or other layer types directly from the viewer,
 but we'll be adding support for this functionality soon as discussed in [#379](https://github.com/napari/napari/issues/379).
 
+If you wish to interact with your open viewer programmatically you can open an IPython
+console via **Window** > **console**.
```

Comment:
```suggestion
console via **Window** > **console** or by clicking the `Show/Hide IPython console` viewer button.
```

Reply:
Thanks, I forgot about that. Done and I've linked to the viewer layer section!